systemd/grive-sync.sh

```diff
@@ -17,11 +17,6 @@ cd ~
 SCRIPT="${0}"
 DIRECTORY=$(systemd-escape --unescape -- "$2")
 
-if [[ -z "$DIRECTORY" ]] || [[ ! -d "$DIRECTORY" ]] ; then
-	echo "Need a directory name in the current users home directory as second argument. Aborting."
```

Comment:
The test is alright (assuming -z tests for "not defined"), but the have should be "need a directors as either absolute path or relative to $HOME"

Reply:
You were right. I took a better look and what exactly was happening and it all about a "/"....